pkg/cmd/orderbook.go

```diff
@@ -29,13 +30,19 @@ var orderbookCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("can't get the symbol from flags: %w", err)
 		}
+		if symbol == "" {
+			return fmt.Errorf("symbol is not found")
+		}
 
 		s := ex.NewStream()
 		s.Subscribe(types.BookChannel, symbol, types.SubscribeOptions{})
+		s.OnBookSnapshot(func(book types.OrderBook) {
```

Comment:
you can use environment to load exchange session

Reply:
@c9s Could you give me an example?

Reply:
oh but that way you will need to fulfill the implementation